graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -2083,7 +2088,23 @@ def _create_split_files_custom_split(
 
         # custom node/edge label
         # create custom mask dataframe for one of the types: train, val, test
-        def process_custom_mask_df(input_df: DataFrame, split_file: CustomSplit, mask_type: str):
+        def process_custom_mask_df(
+            input_df: DataFrame, split_file: CustomSplit, mask_name: str, mask_type: str
+        ):
```

Comment:
We add `mask_name` before the `mask_type` arg, any reason to not add it to as the last argument?

Reply:
Can we also add a docstring for this function?

Reply:
I just think logically mask_name should refer as a static value here, so I put it before the mask_type. In this case we will iterate different mask_type for train/val/test so I put it at last.